.prettierrc

```diff
@@ -1,8 +1,5 @@
 {
-  "semi": true,
   "tabWidth": 2,
   "printWidth": 100,
-  "singleQuote": true,
-  "trailingComma": "all",
-  "jsxBracketSameLine": true
+  "singleQuote": true
```

Comment:
I thought we were giving up on prettierrc completely. Can you explain why we need it?

Reply:
The Stylelint-prettier plugin that formats our CSS depends on it. If we remove the config, Prettier requires double quotes and crashes the pipeline.